common/utils/serialise-url.ts

```diff
@@ -0,0 +1,27 @@
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type ParsedUrlQuery = { [key: string]: any };
```

Comment:
Apologies, I meant the native type from `@types/node` `querystring` that is on the `context`. 
[e.g.](https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAbzgBQIZQM4FMAmBVKAGwEUBXLKATzgF84AzKCEOAcgEdyqMYpgA7AOasA3AFgAUJIDGEfjzgCAbqkLAcALhTps+ImQrUAvIjgBrLJS1Jlq9Vt7latcRNnz4KtZu2ZcBEi5jUwsrNi91VgAac0sAJi0AbVYInGjwuzSAXRcgA)

Reply:
ahh perfect .. thank you